05_book_titles/book.rb

```diff
@@ -1,3 +1,16 @@
+#create class called BOOK
 class Book
-# write your code here
-end
+   attr_reader :title
+    
+  def title=string
+    articles = %w[a an and if or in the of]
+    words = string.split(" ")
+  
+    words.map do |word|
+    word.capitalize! unless articles.include? word
+  end
```

Comment:
Whoops, turns out I forgot how to use RuboCop, so the command I gave you doesn't cover all the cases where indentation could go wrong. 😅  I could give you a _much_ longer command that fixes all the alignment issues, but it's probably better to just not use RuboCop right now.

```suggestion
    words.map do |word|
      word.capitalize! unless articles.include? word
    end
```